torch_frame/datasets/diamond_images.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+import os.path as osp
+
+import pandas as pd
+
+import torch_frame
+from torch_frame.config import ImageEmbedderConfig
+
+
+class DiamondImages(torch_frame.data.Dataset):
+    r"""The `Diamond Images
+    <https://www.kaggle.com/datasets/aayushpurswani/diamond-images-dataset>`_
+    dataset from Kaggle.
+    """
+
+    url = ''  # noqa
+
+    def __init__(
+        self,
+        root: str,
+        col_to_image_embedder_cfg: ImageEmbedderConfig
+        | dict[str, ImageEmbedderConfig],
+        target: str = "shape",
+    ):
+        # path = self.download_url(self.url, root)
+        path = "/Users/zecheng/code/pytorch-frame/web_scraped"
+        csv_path = osp.join(path, "diamond_data.csv")
+        df = pd.read_csv(csv_path)
+        df = df.drop(columns=["stock_number"])
+        path = "/Users/zecheng/code/pytorch-frame"
```

Comment:
:)

Reply:
I need @rusty1s 's help to upload the dataset, probably next week

Reply:
Has this been resolved?

Reply:
I will upload the dataset soon

Reply:
This is not resolved!

Reply:
Just told @rusty1s to upload the dataset. Will be resolved tomorrow.